search-api.md

```diff
@@ -120,49 +126,53 @@ After receiving a request, the remote server can respond in one of two ways:
 
 #### Age of onset
 * *Optional*
-* An [HPO](http://www.human-phenotype-ontology.org/) term identifier (HP:#######) associated with an age interval [as defined by the HPO](http://www.human-phenotype-ontology.org/hpoweb/showterm?id=HP:0011007)
-  * `"HP:0003577"` (Congenital onset)
-    * `"HP:0011460"` (Embryonal onset)
-    * `"HP:0011461"` (Fetal onset)
-  * `"HP:0003623"` (Neonatal onset)
-  * `"HP:0003593"` (Infantile onset)
-  * `"HP:0011463"` (Childhood onset)
-  * `"HP:0003621"` (Juvenile onset)
-  * `"HP:0003581"` (Adult onset)
-    * `"HP:0011462"` (Young adult onset)
-    * `"HP:0003596"` (Middle age onset)
-    * `"HP:0003584"` (Late onset)
+* An age interval, described by:
+  * `id`: an [HPO](http://www.human-phenotype-ontology.org/) term identifier (HP:#######) associated with an [age interval](http://www.human-phenotype-ontology.org/hpoweb/showterm?id=HP:0011007) (***mandatory***)
+    * `"HP:0003577"` (Congenital onset)
+      * `"HP:0011460"` (Embryonal onset)
+      * `"HP:0011461"` (Fetal onset)
+    * `"HP:0003623"` (Neonatal onset)
+    * `"HP:0003593"` (Infantile onset)
+    * `"HP:0011463"` (Childhood onset)
+    * `"HP:0003621"` (Juvenile onset)
+    * `"HP:0003581"` (Adult onset)
+      * `"HP:0011462"` (Young adult onset)
+      * `"HP:0003596"` (Middle age onset)
+      * `"HP:0003584"` (Late onset)
+  * `label`: a human readable description (*optional*)
```

Comment:
No, I don't mean a new field besides label. I meant: define the label as the HPO official name for the term, but as a lax suggestion, since the name depends on the actual HPO version in use.